src/api/fragments/CollectionData.ts

```diff
@@ -14,7 +15,21 @@ export const CollectionData = gql`
     slug
     excerpt
     intro
-    imageUrl
+    image {
+      cachedImages(imageOptions: $imageOptions) {
+        height
+        id
+        url
+        width
+      }
```

Comment:
I tried to create two separate fragments --> `ImageData` and `CachedImageData`. Was running into some compilation errors so thought I'd come back to it after making it work intially.